shm/.gitignore

```
@@ -6,3 +6,4 @@
 !.gitignore
 !*.hpp
 !*.cpp
+!*CMakeLists.txt

```

I think it should be:
```suggestion
!CMakeLists.txt
```